lib/expression/callbacks/standard.ex

```diff
@@ -1434,4 +1434,32 @@ defmodule Expression.Callbacks.Standard do
 
     rem(integer1, integer2)
   end
+
+  @doc """
+  Appends items from one list to another list.
+  """
+  @expression_doc expression: "append([\"A\", \"B\"], [\"C\", \"B\"])",
+                  result: ["A", "B", "C", "B"]
+  def append(ctx, first_list, second_list) do
```

Comment:
Usually a function called `append` will add one element to a list. If this takes 2 lists should we call it `concat` rather?

Reply:
yeah, that's true. But there is also a function called `concatenate` that concatenates strings 😅

Reply:
I see, very unfortunate! :D
Can we maybe have this append function accept both a list or a single element as the second argument?

Reply:
what about `merge`? maybe we can add support for maps too.

Reply:
or maybe just use `concat` as you suggested but also as we can see from [TextIt expressions](https://textit.in/mr/docs/expressions.html#function:concat).

Reply:
either we're going to have to be very disciplined about naming conventions (follow Redis' command examples) or we need to start namespacing into modules.

Reply:
what do you think it's better?
- `append` supporting both, a single element and a list of elements
- `merge` supporting both, lists and maps
- `concat` just for lists

I need to wrap up this because some people are waiting for this right now.

Reply:
if `concatenate` already exists in the expressions we support, should we maybe make it work for lists too in addition to strings?

Reply:
yes, that's also a good option. I like it!

Reply:
@smn what do you think about this? ⬆️

Reply:
I like that idea but keep in mind that concatenate() as it's defined in the spec takes any amount of string arguments and joins them. We'd need to then do the same for lists, so it would be like a combination of Enum.concat and Enum.reduce

Reply:
Ok, I ended up using `append()` which is what two clients were asking for but also to simplify things 👍🏼